money_utils/money_utils.bal

```diff
@@ -13,9 +13,10 @@
 // KIND, either express or implied.  See the License for the
 // specific language governing permissions and limitations
 // under the License.
+
 import wso2/client_stubs as stub;
 
-const map<string> logos = {
+public const map<string> LOGOS = {
```

Comment:
Can/should this be `CURRENCY_SYMBOLS`?

https://en.wikipedia.org/wiki/Currency_symbol

Reply:
Fixed in https://github.com/ballerina-guides/gcp-microservices-demo/pull/65/commits/e746edee6815377481f762a558e106b9e16d715b